### DISCUSSION TITLE
一个可能存在的BUG

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3441] [综合讨论 / General]
想问一下，V2版本是否可以正常提取下面视频的封面？V1的“提取封面”按钮在下面视频的播放页面消失，不知道V2是否存在同样BUG
https://www.bilibili.com/video/av54220

**Comment by Geeyun-JY3** [comment]
V2 能用的

**Reply by Tinhone** [reply]
真好。
伟大的维护人员！（赞美态）
